src/main/java/core/basesyntax/random/ColorSupplier.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.random;
+
+import core.basesyntax.enams.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private String color;
```

Comment:
```suggestion
```
We don't need it as a field, we just use object returned from this method, we don't need to keep it as a state of the class that will change on each method invocation